application-diagram-ui/src/main/resources/Diagram/DiagramTranslations.xml

```diff
@@ -67,6 +67,10 @@ diagram.macro.invalidReference=The specified page is not a diagram.
 diagram.macro.view=View diagram
 diagram.macro.viewNotAllowed=You are not allowed to view this diagram.
 
+# Diagram menu
+diagram.menu.print.title=Print the diagram using an external device when possible, otherwise simply export and save it as PDF.
+diagram.menu.print.label=Print / Export as PDF
```

Comment:
`diagram.editor.menu.print....`
And you can put it in the `Diagram editor` category you have below